source/installguide/management-server/_database.rst

```diff
@@ -296,10 +316,10 @@ same node for MySQL. See `“Install the Database on the Management Server Node
 #. (CentOS and RHEL only; not required on Ubuntu)
 
    .. warning::
-      On RHEL and CentOS, MySQL does not set a root password by default. It is 
-      very strongly recommended that you set a root password as a security 
-      precaution. Run the following command to secure your installation. You 
-      can answer "Y" to all questions except "Disallow root login remotely?". 
+      On RHEL and CentOS, MySQL does not set a root password by default. It is
```

Comment:
This is not true anymore with the latest 5.7.latest and 8.x, ( a random password is generated), this but doesn't matter... leave it as it is